packages/lib-classifier/src/store/FieldGuideStore.spec.js

```diff
@@ -48,35 +47,47 @@ describe('Model > FieldGuideStore', function () {
       })
   }
 
+  function setupStores (clientStub) {
+    rootStore = RootStore.create({
```

Comment:
I think this should return the new store, like we're doing with the other store tests.